### PR TITLE
[SID-1368] Record logout and person identified analytics events, make analytics opt-out

### PR DESCRIPTION
**Pull request by jake-slashid**
Dependencies:
- https://github.com/slashid/ng-evangelion/pull/1200
- https://github.com/slashid/ng-evangelion/pull/1204

## Description

[YouTrack issue](https://todo.irdcorp.dev/issue/SID-1368)

- Any time the user is stored record this as a "person identified" (login, parse from token storage)
- When the user explicitly logs out, record this as a "person logged out"
- Make analytics on by default

### CODE REVIEW

File: packages/react/package.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
   },
   "devDependencies": {
     "@faker-js/faker": "^8.0.2",
-    "@slashid/slashid": "3.15.0",
+    "@slashid/slashid": "file:.yalc/@slashid/slashid",
```
Comment (paulo-raca) [inline packages/react/package.json:65]: How does it work? The version is referenced in a gitignored file? :thinking:

**Review by paulo-raca — APPROVED** [review]

File: packages/react/package.json (hunk 1 above)
Comment (jake-slashid) [reply packages/react/package.json:65]: This is just a placeholder, but it's a tool for publishing packages on local machine for development

I'll change this to the actual version once its published

https://github.com/wclr/yalc

Comment (jake-slashid) [reply packages/react/package.json]: done

File: packages/react/package.json [hunk 2]
```diff
@@ -62,7 +62,7 @@
   },
   "devDependencies": {
     "@faker-js/faker": "^8.0.2",
-    "@slashid/slashid": "3.15.0",
+    "@slashid/slashid": "^3.16.0",
```
Comment (ikovic) [inline packages/react/package.json:65]: Should we go with a fixed version? Just to prevent surprises from happening 😅

Comment (jake-slashid) [reply packages/react/package.json]: Done!

**Pull request merged**